docs/guides/securing-apps/token-exchange.adoc

```diff
@@ -22,6 +22,10 @@ The standard token exchange supports only use-case (1). The legacy token exchang
 backwards compatible with future {project_name} versions. You can also enable both token exchange features and use them together. For example, you could use both internal-internal exchange
 provided by V2 together with other use cases that are supported by V1. For more details, see this <<_standard-token-exchange-comparison,token exchange comparison>>.
 
+NOTE: If you still need legacy token exchange feature, you also need link:{adminguide_link}#_fine_grained_permissions_v1[Fine-grained admin permissions version 1] enabled (FGAP:v1) because
```

Comment:
```suggestion
NOTE: If you still need the legacy token exchange feature, you also need the link:{adminguide_link}#_fine_grained_permissions_v1[Fine-grained admin permissions version 1] enabled (FGAP:v1) because
```